engine/audio/private/audio_system.cpp

```diff
@@ -0,0 +1,159 @@
+#include "audio_system.hpp"
+
+#include "audio_emitter_component.hpp"
+#include "audio_listener_component.hpp"
+#include "audio_module.hpp"
+#include "components/rigidbody_component.hpp"
+#include "components/transform_component.hpp"
+#include "components/transform_helpers.hpp"
+#include "ecs_module.hpp"
+#include "fmod_debug.hpp"
+
+#include <fmod.h>
+#include <ranges>
+
+#include <tracy/Tracy.hpp>
+
+AudioSystem::AudioSystem(ECSModule& ecs, AudioModule& audioModule)
+    : _ecs(ecs)
+    , _audioModule(audioModule)
+{
+}
+void AudioSystem::Update(ECSModule& ecs, MAYBE_UNUSED float dt)
+{
+    ZoneScoped;
+    const auto& listenerView = ecs.GetRegistry().view<AudioListenerComponent>();
```

Comment:
Having an Audio Listener component might be a little useless since only one entity can really have this component. Either way, it works for tagging the player entity

Reply:
It is indeed just a tag for the audio system to know what entity is listening, it could possibly later be expanded